.buildkite/pipeline-release.yml

```diff
@@ -0,0 +1,24 @@
+steps:
+  - group: release
+    steps:
+
+      - label: "k8s manifests"
+        if: |
+          ( build.branch == "main" && build.source != "schedule" )
+          || build.tag != null
+          || build.message =~ /^buildkite test .*release.*/
+        commands:
+          - buildkite-agent artifact download 'config/*.yaml' config/
+          - .buildkite/scripts/build/setenv.sh
+          - .buildkite/scripts/release/k8s-manifests.sh
+
+      - label: "helm release"
+        commands:
+          - buildkite-agent pipeline upload .buildkite/pipeline-release-helm.yml
+
+      - label: "redhat operatorhub"
+        if: |
+          build.message =~ /^buildkite test .*release.*/
+          || build.message == "release eck on operatorhub"
```

Comment:
```suggestion
          || build.message =~ /^release eck .*operatorhub.*/
```

Reply:
I'm not sure to understand the purpose. Is it to be able to `publish eck operatorhub and eck-operator helm charts` for example at the same time?

Reply:
Feels like "release eck operatorhub" should suffice for the message, so this includes "release eck on|in operatorhub". Hope that makes sense